src/main/java/seedu/duke/Ui.java

```diff
@@ -2,21 +2,38 @@
 
 import java.util.Scanner;
 
+/**
+ * This class handles all UI-related functions.
+ */
 public class Ui {
     private Scanner userInputScanner;
 
+    /**
+     * Initialize a UI handler.
+     */
     public Ui() {
         userInputScanner = new Scanner(System.in);
     }
 
+    /**
+     * Returns user input as a String.
+     * @return user input
+     */
     public String readInput() {
         return userInputScanner.nextLine();
     }
 
+    /**
+     * Closes scanner.
+     */
     public void closeScanner() {
         userInputScanner.close();
     }
 
+    /**
+     * Prints the String specified in @param.
+     * @param printstr String to be printed
+     */
     public void printString(String printstr) {
```

Comment:
I don't think we should use str since it is an abbreviation, the argument could just be called `string`.
Alternatively, you could also rename the method to `printMessage` and the argument to `message`.